learnosity_sdk/request.py

```diff
@@ -30,9 +33,9 @@ def __init__(
             self, service, security, secret,
             request=None, action=None):
         self.service = service
-        self.security = security
+        self.security = security.copy()
```

Comment:
Because the recursive data api fetch reused the request object, but changes the "next" value to point to the next page. It was in implementing that that I found this bug.